meson_options.txt

```diff
@@ -2,3 +2,8 @@ option('preload_libasan',
        type: 'string',
        value: 'no',
        description: 'Path to libasan.so to preload')
+
+option('enable_explicit_EC_test',
```

Comment:
Not sure if this needs to be generic, but meson provides a `feature` [option](https://mesonbuild.com/Build-options.html#features) type for enabled/disabled kind of options.

Reply:
Not sure either, I'm guessing it would be more useful if we had a dependency in this case, but since this is just a on-and-off switch I don't see the need for it.